packages/dose/dose.3.4.1-beta1/url

```diff
@@ -0,0 +1,2 @@
+https://gforge.inria.fr/frs/download.php/file/35382/dose3-4.1-beta1.tar.gz
```

Comment:
This should be:

`archive: "https://gforge.inria.fr/frs/download.php/file/35382/dose3-4.1-beta1.tar.gz"`